ush/prepobs_syndata.sh

```diff
@@ -155,18 +155,14 @@ TIMEIT=${TIMEIT:-""}
 [ -s $DATA/time ] && TIMEIT="$DATA/time -p"
 
 SITE=${SITE:-""}
-sys_tp=${sys_tp:-$(getsystem.pl -tp)}
+sys_tp=${sys_tp:-$(getsystem -tp)}
```

Comment:
`getsystem -tp` is not valid on WC2. You need to use different options or none at all.